app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -77,6 +77,8 @@ public void setupViewPager(ViewPager viewPager) {
     public boolean onCreateOptionsMenu(Menu menu) {
         menu.clear();
         getMenuInflater().inflate(R.menu.menu_new_contest, menu);
+        MenuItem nextItem = menu.findItem(R.id.action_next);
+        nextItem.setVisible(nextVisible);
```

Comment:
close, but we're not actually trying to hide the arrow right? We just want to show a grey one and prevent the user from moving forward.